test/WebSites/FormatterWebSite/Controllers/InputFormatterController.cs

```diff
@@ -9,35 +9,6 @@ namespace FormatterWebSite.Controllers
 {
     public class InputFormatterController : Controller
     {
-        [HttpPost]
-        public object ActionHandlesError([FromBody] DummyClass dummy)
-        {
-            if (!ModelState.IsValid)
-            {
-                // Body model binder normally reports errors for parameters using the empty name.
-                var parameterBindingErrors = ModelState["dummy"]?.Errors ?? ModelState[string.Empty]?.Errors;
-                if (parameterBindingErrors != null && parameterBindingErrors.Count != 0)
-                {
-                    return new ErrorInfo
-                    {
-                        ActionName = "ActionHandlesError",
-                        ParameterName = "dummy",
-                        Errors = parameterBindingErrors.Select(x => x.ErrorMessage).ToList(),
-                        Source = "action"
-                    };
-                }
-            }
-
-            return dummy;
-        }
-
-        [HttpPost]
-        [ValidateBodyParameter]
-        public object ActionFilterHandlesError([FromBody] DummyClass dummy)
-        {
-            return dummy;
-        }
-
```

Comment:
I don't think so, the behavior for these two actions is covered by other tests already, these tests only check that there's a value in the model state when content-type is invalid and that you can handle it either on the action or using a filter. Given that we have a filter that does this by default now and that we have test to cover this behavior, I don't see any reason to keep these around